algebird-core/src/main/scala/com/twitter/algebird/HyperLogLog.scala

```diff
@@ -43,6 +43,8 @@ object HyperLogLog {
   /* Size of the hash in bits */
   val hashSize = 128
 
+  val powersOfNegativeTwo: Array[Double] = 0.until(hashSize).map{ i => math.pow(2.0, -i) }.toArray
```

Comment:
You're always setting aside at least 4 bits for the bucket, so I think the max is actually 124 - this should be slightly more than needed.